src/core/lib/surface/call.h

```diff
@@ -102,7 +102,8 @@ void grpc_call_context_set(grpc_call* call, grpc_context_index elem,
 void* grpc_call_context_get(grpc_call* call, grpc_context_index elem);
 
 #define GRPC_CALL_LOG_BATCH(sev, call, ops, nops, tag) \
-  if (grpc_api_trace.enabled()) grpc_call_log_batch(sev, call, ops, nops, tag)
+  if (GRPC_TRACE_FLAG_ENABLED(grpc_api_trace))         \
+  grpc_call_log_batch(sev, call, ops, nops, tag)
```

Comment:
In the spirit of avoiding programming errors, perhaps this should also be wrapped in something?

Reply:
Thank you for noting this. Sure converted this and 4 others over.